### ISSUE TITLE
Feature request: afterLoadRelations to contain the original loaded relations

### ISSUE THREAD

**Issue by elado** [js-data/js-data#254]
```js
const Group = store.defineModel('group', {
  // ...
  afterLoadRelations(resource, instance, relations) {
    // relations would be ['organization', 'members']
  }
})

// ...

Group.loadRelations(group, ['organization', 'members'])
```

**Comment by jmdobry** [comment]
I'd have to find a way to implement this in a non-breaking way. We can't break things for someone who currently has something like this:

```js
  afterLoadRelations: function (Resource, instance, cb) {
    // ...
    cb(null, instance);
  },
```

**Comment by elado** [comment]
Because of the `promisify`? Good point, I was starting to write a PR.
The use case is when I want to have specific code to load some relationships (many to many stuff, using `computed` properties) but in different cases I want to load different relationships.

**Comment by jmdobry** [comment]
This has been implemented in 3.0.0-alpha.x
